pkg/agent/openflow/fields.go

```diff
@@ -60,6 +60,8 @@ var (
 	NotRewriteMACRegMark = binding.NewOneBitZeroRegMark(0, 9, "NotRewriteMAC")
 	// reg0[10]: Mark to indicate the packet is denied(Drop/Reject).
 	CnpDenyRegMark = binding.NewOneBitRegMark(0, 10, "CNPDeny")
+	// reg0[18]: Mark to indicate the packet is allowed
+	CnpAllowRegmark = binding.NewOneBitRegMark(0, 18, "CNPAllow")
```

Comment:
What is the difference from field `DispositionAllowRegMark`?

Reply:
removed as the rule can work without such a regmark